### DISCUSSION TITLE
Bad schedule will not throw error message

### DISCUSSION THREAD

**Discussion by ogmueller** [PremoWeb/chadburn#76] [General]
I entered an incorrect schedule "@every 1d", because I thought that "d" is also allowed. I would have expected chadburn to report an error, when starting up/reading in the labels, but it doesn't.

Somehow chadburn is "reading/registering" the cron anyway, because once I fixed the problem, the log said this:
```
chadburn                | scheduler.go:94 ▶ NOTICE Job deregistered (will not fire again) "optimize" - "php -dmemory_limit=-1 /web/app/console.php index:optimize" - "@every 1d" - ID: 0
```

It wasn't showing the registration, but the deregistration of the cron.

An error message would be very helpful in the logs, so those kind of problems can be identified easier.

PS: It would be great, if the log would also add information about the container (e.g. name or ID)

**Comment by sebbi08** [comment]
Same is for bad crons like this "0 0 4 * * *"

**Comment by maietta** [comment]
Sorry for the really long delay, @ogmueller. Don't know how I missed this. I agree, this should be raising an error.

Also yes, @sebbi08, agreed.

This will get escalated but there is no E.T.A.

**Comment by calvinbui** [comment]
> Same is for bad crons like this "0 0 4 * * *"

this is a good cron according to [the docs](https://github.com/PremoWeb/chadburn/blob/main/README.md?plain=1#L35) but doesn't work.

If you exclude the seconds, `0 4 * * *` it will work.
